tests/wpt/metadata/offscreen-canvas/compositing/2d.composite.canvas.copy.html.ini

```diff
@@ -0,0 +1,2 @@
+[2d.composite.canvas.copy.html]
+  expected: CRASH
```

Comment:
If you remove this file and run this test, what is the output? We should figure out why it's crashing.